samples/bluetooth/direct_test_mode/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Use an additional timer for Anomaly 172
+CONFIG_NRFX_TIMER3=y
```

Comment:
The nrf21540dk_nrf52840 also needs anomaly timer

Reply:
thanks, fixed it